README.md

```diff
@@ -141,3 +141,31 @@ file `~/.oauth2l.token`.
 ```
 $ oauth2l reset
 ```
+
+## Options
+
+### `--json`
+
+Use a JSON key file downloaded from
```

Comment:
Specifies an OAuth credential file, either OAuth client ID or Service Account key, to start the OAuth flow. You can download the file from ...

Please link to the credential page, not the main page.